CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.6)
+project(Pokopomclion)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_BUILD_TYPE Debug)
```

Comment:
Would setting this to Release make it the default build type on Visual Studio? That would be preferable.